_sass/minimal-mistakes/skins/_rte.scss

```diff
@@ -6,13 +6,14 @@
 $background-color: #eeeeee !default;
 $text-color: #222831 !default;
 $muted-text-color: #393e46 !default;
-$primary-color: #00A6D9 !default;
+$primary-color: #00a6d9 !default;
 $border-color: mix(#fff, #393e46, 75%) !default;
 $footer-background-color: $primary-color !default;
 $link-color: #393e46 !default;
 $masthead-link-color: $text-color !default;
 $masthead-link-color-hover: $text-color !default;
 $navicon-link-color-hover: mix(#fff, $text-color, 80%) !default;
+@import "minimal-mistakes"; // import minimal-mistakes mixines
```

Comment:
this is already in minimal mistakes' ["main.scss"](https://github.com/mmistakes/minimal-mistakes/blob/master/assets/css/main.scss) , is it necessary here too ?